src/libs/abstraction/web2/domain.ts

```
@@ -0,0 +1,92 @@
+    constructor() {
+        super()
+    }
+
+    async readData(
+        proofUrl: string,
+    ): Promise<{ message: string; type: SigningAlgorithm; signature: string }> {
+        this.verifyProofFormat(proofUrl, "domain")
+
+        let body: string
+        try {
+            ;({ body } = await fetchDomainProof(proofUrl))
```

ESLint

[error] Unnecessary semicolon.

(no-extra-semi)